### PR TITLE
Give user suggestion about new-app on new-project

### PR DESCRIPTION
**Pull request by smarterclayton**
Don't assume they know about it.

[test]

### CODE REVIEW

**Comment by smarterclayton** [comment]
```
$ oc new-project bazz
Now using project "bazz" on server "https://10.1.2.2:8443".

You can add applications to this project with the 'new-app' command. For example, try:

    $ oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-hello-world.git

to build a new hello-world application in Ruby.
```

**Comment by smarterclayton** [comment]
or @csrwng review please

File: pkg/cmd/cli/cmd/request_project.go [hunk 1]
```diff
@@ -121,5 +124,13 @@ func (o *NewProjectOptions) Run() error {
 		}
 	}
 
+	fmt.Fprintf(o.Out, `
+You can add applications to this project with the 'new-app' command. For example, try:
```
Comment (fabianofranz) [inline pkg/cmd/cli/cmd/request_project.go]: Not required but could also make a reference to `oc new-app -h`.

File: pkg/cmd/cli/cmd/request_project.go [hunk 2]
```diff
@@ -45,9 +46,11 @@ After your project is created it will be made your default project in your confi
   $ %[1]s web-team-dev --display-name="Web Team Development" --description="Development project for the web team."`
 )
 
-func NewCmdRequestProject(name, fullName, ocLoginName, ocProjectName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+func NewCmdRequestProject(baseName, name, ocLoginName, ocProjectName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
 	options := &NewProjectOptions{}
 	options.Out = out
+	options.Name = baseName
```
Comment (fabianofranz) [inline pkg/cmd/cli/cmd/request_project.go]: Where are you using `options.Name`?

File: pkg/cmd/cli/cmd/request_project.go [hunk 3]
```diff
@@ -121,5 +124,13 @@ func (o *NewProjectOptions) Run() error {
 		}
 	}
 
+	fmt.Fprintf(o.Out, `
+You can add applications to this project with the 'new-app' command. For example, try:
+
+    $ %[1]s new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-hello-world.git
+
+to build a new hello-world application in Ruby.
+`, o.Name)
```
Comment (fabianofranz) [inline pkg/cmd/cli/cmd/request_project.go]: Got it.

**Comment by fabianofranz** [comment]
LGTM

**Comment by csrwng** [comment]
LGTM

**Comment by csrwng** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**